jans-cedarling/bindings/cedarling_go/README.md

```
@@ -267,6 +267,68 @@ logs := instance.GetLogsByTag("info")
+    // From a local JSON file
+    "CEDARLING_POLICY_STORE_LOCAL_FN": "/path/to/policy-store.json",
+    // Or from Lock Server URI
+    "CEDARLING_POLICY_STORE_URI": "https://lock-server.example.com/policy-store",
+}
+```
+
+#### New Directory-Based Format
+
+Policy stores can be structured as directories with human-readable Cedar files:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)